indico/modules/designer/client/js/index.js

```diff
@@ -240,11 +244,28 @@ import {$T} from 'indico/utils/i18n';
     var itemStyles = _.filter([item.bold ? 'bold' : null, item.italic ? 'italic' : null]);
 
     // Change the selectors so that they match the properties of the item
+    $('#text-color-selector').val(item.color);
+    var textColorPreview = $('#text-color-preview');
+    if (textColorPreview.hasClass('no-value') && item.color.length > 1) {
```

Comment:
No need for that hasClass check; removing a class that isn't there is just a no-op. And I think you can greatly simplify it like this:

```
textColorPreview.toggleClass('no-value', !item.color);
```